scripts/build/dependencies-npm.sh

```diff
@@ -88,7 +88,9 @@ else
   export npm_config_arch=$ARGV_ARCHITECTURE
 fi
 
-INSTALL_OPTS=""
+# Force `electron-prebuilt` to download the right binary
+# See: https://github.com/electron-userland/electron-prebuilt
+INSTALL_OPTS="--arch=$npm_config_arch"
```

Comment:
Indeed, I duplicated this for debugging purposes (to see if manually passing `--arch` made any difference) and forgot to remove. Done!